readme.md

```diff
@@ -1080,7 +1080,15 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | -----: |
 |   2  |        8+ | 30m around|
 
+### [Say What? Shiok Food Card Game](https://www.saywhatwithfriends.com/shop/say-what-singapores-shiok-food-card-game)
 
+> Love Singapore’s Local Food? This Food Card Game contains 50 local dishes for you to guess and the winner will be the one who can guess the most.
+
+![Say What? Shiok Food Card Game](https://images.squarespace-cdn.com/content/v1/5649b2dbe4b0142472e867bc/1573954498990-B1NESNC79U51TJNXQEBB/SG+FOOD+GAME+FRONT.JPG?format=1000w)
```

Comment:
This image is far too large, [as is visible in the PR preview build](https://awesome-board-games-git-fork-xinyitay-adding-shi-dc53b1-edm00se.vercel.app/#say-what-shiok-food-card-game).

Please follow [the formatting guide](https://github.com/edm00se/awesome-board-games/blob/main/formatting.md) and make use of an appropriately sized image.

Reply:
```suggestion
![Say What? Shiok Food Card Game](https://images.squarespace-cdn.com/content/v1/5649b2dbe4b0142472e867bc/1573954498990-B1NESNC79U51TJNXQEBB/SG+FOOD+GAME+FRONT.JPG?format=250w)

I found that with the format parameter that the image link can return a more reasonable image size. If you accept this code suggestion that should meet the needs of this PR.
```